<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-27T15:20:17 IST" name="Default suite" finished-at="2021-08-27T15:21:11 IST" duration-ms="54033">
    <groups>
      <group name="smokeTest">
        <method signature="TC_21.Opportunity()[pri:0, instance:Opportunity.TC_21@21e360a]" name="Opportunity" class="Opportunity.TC_21"/>
      </group> <!-- smokeTest -->
    </groups>
    <test started-at="2021-08-27T15:20:17 IST" name="Default test" finished-at="2021-08-27T15:21:11 IST" duration-ms="54033">
      <class name="Opportunity.TC_21">
        <test-method is-config="true" signature="jdbc_report()[pri:0, instance:Opportunity.TC_21@21e360a]" started-at="2021-08-27T15:20:17 IST" name="jdbc_report" finished-at="2021-08-27T15:20:17 IST" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jdbc_report -->
        <test-method is-config="true" signature="launch_browser()[pri:0, instance:Opportunity.TC_21@21e360a]" started-at="2021-08-27T15:20:17 IST" name="launch_browser" finished-at="2021-08-27T15:20:22 IST" duration-ms="5178" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch_browser -->
        <test-method is-config="true" signature="login_page()[pri:0, instance:Opportunity.TC_21@21e360a]" started-at="2021-08-27T15:20:22 IST" name="login_page" finished-at="2021-08-27T15:20:29 IST" duration-ms="6860" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_page -->
        <test-method signature="Opportunity()[pri:0, instance:Opportunity.TC_21@21e360a]" started-at="2021-08-27T15:20:29 IST" name="Opportunity" finished-at="2021-08-27T15:21:08 IST" duration-ms="39600" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [[ POT99 ] das -  Opportunity Information] but found [[[ChromeDriver: chrome on WINDOWS (05c5dafddb96d41c162faf066cc5b87c)] -> xpath: //div[@class='small']/../../..//span[@class='dvHeaderText']]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [[ POT99 ] das -  Opportunity Information] but found [[[ChromeDriver: chrome on WINDOWS (05c5dafddb96d41c162faf066cc5b87c)] -> xpath: //div[@class='small']/../../..//span[@class='dvHeaderText']]]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at Opportunity.TC_21.Opportunity(TC_21.java:49)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Opportunity -->
        <test-method is-config="true" signature="logout_page()[pri:0, instance:Opportunity.TC_21@21e360a]" started-at="2021-08-27T15:21:08 IST" name="logout_page" finished-at="2021-08-27T15:21:11 IST" duration-ms="2332" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout_page -->
        <test-method is-config="true" signature="jdbc_report2()[pri:0, instance:Opportunity.TC_21@21e360a]" started-at="2021-08-27T15:21:11 IST" name="jdbc_report2" finished-at="2021-08-27T15:21:11 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jdbc_report2 -->
      </class> <!-- Opportunity.TC_21 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
